griptape/utils/chat.py

```diff
@@ -10,24 +11,30 @@
 class Chat:
     structure: Structure = field()
     exit_keywords: list[str] = field(default=["exit"], kw_only=True)
-    exiting_text: str = field(default="exiting...", kw_only=True)
-    processing_text: str = field(default="processing...", kw_only=True)
+    exiting_text: str = field(default="Exiting...", kw_only=True)
+    processing_text: str = field(default="Thinking...", kw_only=True)
     intro_text: Optional[str] = field(default=None, kw_only=True)
-    prompt_prefix: str = field(default="Q: ", kw_only=True)
-    response_prefix: str = field(default="A: ", kw_only=True)
-    output_fn: Callable[[str], None] = field(default=Factory(lambda: print), kw_only=True)
+    prompt_prefix: str = field(default="User: ", kw_only=True)
+    response_prefix: str = field(default="Assistant: ", kw_only=True)
+    output_fn: Callable[[str], None] = field(default=lambda x: print(x, end="", flush=True), kw_only=True)
```

Comment:
Went back and forth on whether there should be two output functions. One for regular printing, one for stream printing. This feels kind of hacky, open to feedback.